src/server/game/gamemanager.js

```diff
@@ -770,7 +770,7 @@ const gamemanager = (function() {
 
         if (isGameOver(game)) { // Resync them to the game because they did not see the game conclusion.
             console.error("Player tried to resign game when the game is already over!")
-            ws.metadata.sendmessage(ws, 'general', 'notify', "Can't resign game, it's already over.")
+            ws.metadata.sendmessage(ws, 'general', 'printerror', "Can't resign game, it's already over.")
```

Comment:
I'm in favor of this getting it's own code for the 'notify' action, because otherwise the player will have thought they resigned, not knowing it ended a split second before then.